### PR TITLE
fixed '_x' latex issue

### PR DESCRIPTION
**Pull request by rijusougata13**
Leading underscores and `latex()` bug fixed
#### References to other Issues or PRs

fixed  #14718, replaces  #19984

#### Brief description of what is fixed or changed

i have replaced the leading underscore with blank
#### Other comments

NONE
#### Release Notes

*  printing
   *   fixed leading underscored bug

### CODE REVIEW

**Comment by rijusougata13** [comment]
this can remove the '_' from begining

**Review by eric-wieser — CHANGES_REQUESTED** [review]
Please do not modify the `latex_escape` function. 
Its purpose is to escape latex, such that if you compile `latex_escape("a string_with_junk\%{")` to a pdf, you'll see exactly `a string_with_junk\%{` in the pdf.

The issue you are referencing is about `_print_Symbol` and friends, which do not even call this function.

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -131,6 +131,7 @@ def latex_escape(s):
     s = s.replace('\\', r'\textbackslash')
     for c in '&%$#_{}':
         s = s.replace(c, '\\' + c)
+    s=s.replace('_','')
```
[the remark this answers is not in the document]
Comment (czgdp1807) [reply sympy/printing/latex.py:134]:
```suggestion
    s = s.replace('_', '')    
```

File: sympy/printing/latex.py [hunk 2]
```diff
@@ -131,6 +131,7 @@ def latex_escape(s):
     s = s.replace('\\', r'\textbackslash')
     for c in '&%$#_{}':
         s = s.replace(c, '\\' + c)
+    s = s.replace('_', '')
```
Comment (eric-wieser) [inline sympy/printing/latex.py:134]:
```suggestion
    s = s.replace('_', '')
```

File: sympy/printing/latex.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rijusougata13) [reply sympy/printing/latex.py]: i didn't get this you told me not to change that line .what should i do now?

[the remark this answers is not in the document]
Comment (eric-wieser) [reply sympy/printing/latex.py]: You're right, my point here was just that if you fix the whitespace that causes the tests to not run at all, you'll see what the problem is:
```
_____________ sympy/printing/tests/test_latex.py:test_latex_escape _____________
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/sympy-1.7.dev0-py3.8.egg/sympy/testing/runtests.py", line 1359, in _timeout
    function()
  File "/home/travis/virtualenv/python3.8.0/lib/python3.8/site-packages/sympy-1.7.dev0-py3.8.egg/sympy/printing/tests/test_latex.py", line 2661, in test_latex_escape
    assert latex_escape(r"~^\&%$#_{}") == "".join([
AssertionError
```

The failing test is this one:

https://github.com/sympy/sympy/blob/e720a533224009d474f80e66a2e8327e3197b490/sympy/printing/tests/test_latex.py#L2660-L2672

> what should i do now?

You should read through the discussion in the original issue, and have a look at the proposed solution in #15939. The direction you're exploring at the moment is not the right one at all.

[the remark this answers is not in the document]
Comment (rijusougata13) [reply sympy/printing/latex.py]: oh ok ..sorry for being a beginner this github commits going over my head. i am confused wht to do next..i have to learn more about this githubs...thank you for helping me out

**Comment by czgdp1807** [comment]
@rijusougata13 Would you like to conclude this PR? The tests are failing, please fix them if so, otherwise do not close this PR. Thanks for your contributions.

**Comment by rijusougata13** [comment]
ok i am closing this pr

**Pull request closed without merging**